File: 05NextRound/next_round.py
count = 0
list = []
n = int(input())
k = int(input())

while n > 0:
    a = int(input())
    list.append(a)
    n -= 1

for student_marks in list:
    if student_marks > 0 and student_marks >= list[k-1]:
        count += 1

print(count)

File: 05NextRound/next_round.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n, k;
    int count = 0;
    cin >> n >> k;
    int arr[n];
    for (int i = 0; i < sizeof(arr) / 4; i++)
        cin >> arr[i];

    for (int i = 0; i < sizeof(arr) / 4; i++)
    {
        if (arr[i] > 0 && arr[i] >= arr[k - 1])
        {
            count++;
        }
    }

    cout << count;
    return 0;
}